views/group/new.pug

```diff
@@ -75,6 +82,9 @@ block scripts
   script(src='https://cdn.jsdelivr.net/npm/@fullcalendar/daygrid@4.3.0/main.min.js')
   script(src='https://cdn.jsdelivr.net/npm/@fullcalendar/timegrid@4.3.0/main.min.js')
   script(src="/js/picker.js")
+  if isEditing
+    script const isEditing = true
```

Comment:
Why are you redeclaring `isEditing` here?

Reply:
The exterior javascript files (like group/create.js) does not get the variables. But I'll see around if there's any clever solution for this.